src/main/resources/jmetemplate/[IF=MULTIPLATFORM]/build.gradle.jmetemplate

```diff
@@ -12,6 +12,14 @@ buildscript {
 
 }
 
+allprojects {
+    repositories {
+        google()
+        mavenCentral()
+        jcenter() // Warning: this repository is going to shut down soon
```

Comment:
Does this really need jcenter? Isn't jcenter completely gone?

Reply:
Honestly I didn't even realize about it, I just moved the existing repositories from settings.gradle (dependencyResolutionManagement section) to build.gradle (allprojects section) to avoid issues with MOE gradle plugin. It adds repositories to the inner project and configured using dependencyResolutionManagement made it all fail.

I think you're right, as far as I remember jcenter service stopped in 2021. Let me double check everything works as expected without jcenter and I'll update the PR removing it.

Thanks

Reply:
Just checked and removed jcenter from all projects